roles/cifmw_cephadm/tasks/cephnfs.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# waiting for https://github.com/ceph/ceph/pull/53108
```

Comment:
Is this still comment still relevant? That change was closed in favor of https://github.com/ceph/ceph/pull/53124, which was merged on August 31st, 2023.

Reply:
@katarimanojk yeah we can remove L17, and rebase the patch against main